tests/qm-ipc/main.fmf

```diff
@@ -0,0 +1,7 @@
+summary: Test plan for qm ipc
+test: ./test_ipc.sh
+duration: 10m
+tag: qm-ipc
```

Comment:
I do not see any plan, include that tag
Where do you think it should fit?

Reply:
I thought it's running by default.